unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotsView.cs

```diff
@@ -66,6 +77,21 @@ public void SetSlotsAsHidden(string[] slotsToHide, string hiddenBy)
                     avatarSlots[slotToHide].SetIsHidden(true, hiddenBy);
         }
 
+        private void RecalculateHideList()
+        {
+            foreach (var slot in avatarSlots)
```

Comment:
I didn't understand sorry :(

Reply:
something like this:
```
foreach ((string category, IAvatarSlotComponentView slot) in avatarSlots)
    slot.ApplyHideList(avatarSlots);
....

IAvatarSlotComponentView.ApplyHideList(Dictionary<string, IAvatarSlotComponentView> avatarSlots)
{
    if (hideList == null) return;

    foreach (string s in hideList)
    {
        if (avatarSlots.ContainsKey(s))
            avatarSlots[s].SetIsHidden(true, slot.Key);
    }
}
```
you utilize your internal hideList instead of exposing it